Updated UI images and made copy edits (#16685)
DataDog/documentation: content/en/metrics/advanced-filtering.md

## Overview

When using the Metrics Explorer, monitors, or dashboards to query metrics data, you can filter the data to narrow the scope of the timeseries returned. Any metric can be filtered by tag(s) using the **from** field to the right of the metric. 

You can also perform advanced filtering with Boolean or Wildcard tag value filters. For queries outside of metrics data such as logs, traces, Network Monitoring, Real User Monitoring, Synthetics, or Security, see the [Log Search Syntax][1] documentation for configuration.

> To use the examples presented, copy and paste into your graph editor to see the the query in the UI by clicking on the code icon `</>`.

Reviewer: Reworded a little, let me know if it's not what you meant!

```suggestion
To use the examples below, click the code icon `</>` to see the query editor in the UI, and then copy and paste the query example into the query editor.
```

> {{< img src="metrics/advanced-filtering/graph_editor_code_option.mp4" alt="Click code icon to see the raw query" video=true >}}

Reviewer: I would recommend moving this to the Boolean filtered queries section, right before the first example.

Reviewer: @estherk15 just checking if you plan on moving this or will leave here?

Author: Thanks @maycmlee , I moved it from the overview to the Boolean filtered query examples. I put it below the example to match up the format of the other images. If you think it would look better above let me know!

Reviewer: >  I moved it from the overview to the Boolean filtered query examples.

Strange, I don't see the change: https://a.cl.ly/geuOQOJN

Author: Ooooh, I only moved the video. I'll update it!

**Note**: Prefix and suffix wildcard matching in the same filter is not supported.

### Wildcard filtered query examples

```
avg:system.disk.in_use{!device:/dev/loop*} by {device}
```

> {{< img src="metrics/advanced-filtering/wildcard_prefix.png" alt="Wildcard used as prefix" style="width:100%;" >}}

Reviewer: This image doesn't match the query example. I think it needs to be swapped with `metrics/advanced-filtering/wildcard_suffix.png`

Author: Thanks for catching this!

```
sum:kubernetes.pods.running{service:*-canary} by {service}
```

> {{< img src="metrics/advanced-filtering/wildcard_suffix.png" alt="Wildcard used as suffix" style="width:100%;" >}}

Reviewer: Image doesn't match the query example.
